content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -18,19 +17,28 @@ Kubernetes DNS schedules a DNS Pod and Service on the cluster, and configures
 the kubelets to tell individual containers to use the DNS Service's IP to
 resolve DNS names.
 
-### What things get DNS names?
-
 Every Service defined in the cluster (including the DNS server itself) is
-assigned a DNS name.  By default, a client Pod's DNS search list will
-include the Pod's own namespace and the cluster's default domain.  This is best
-illustrated by example:
+assigned a DNS name. By default, a client Pod's DNS search list will
+include the Pod's own namespace and the cluster's default domain. 
+
+### Namespaces of Services 
+
+DNS queries for services change based on namespace of the pod. DNS queries that don't specifiy a namespace are limited to the pod's namespace. Services in other namespaces may be accessed by specifying the namespace. 
+
+For example, consider a pod in a `test` namespace. A `data` service is in the `prod` namespace. 
+
+A query for `data` returns no results, because it uses the pod's `test` namespace. 
```

Comment:
How about:
```suggestion
If that pod makes a DNS query to resolve `data` there are no results
found, because it uses the pod's _test_ namespace. The actual DNS name
queried is similar to `data.test.cluster.local.` (the `cluster.local.` part
is Kubernetes' default zone, but you can configure it to be something different).

`data.prod.cluster.local.` does exist in DNS. A pod in the _test_ namespace
can look up `data.prod` or `data.prod.cluster.local.` and resolve either name OK.
```

Reply:
I added additional description of DNS queries based on this. Thanks!

Reply:
> A query for `data` returns no results, because it uses the pod's `test` namespace. 

This actually depends, right? If the query is raised from within the `prod` namespace, the query will succeed, right? Please be precise when describing what works and what doesn't work.

Reply:
this is exactly what i'm trying to clarify with the PR

How does this sentence sound?

> DNS queries for services change based on namespace of the pod. DNS queries that don't specify a namespace are limited to the pod's namespace. Services in other namespaces may be accessed by specifying the namespace.

Reply:
I also think the section, as a whole, makes this clear 

https://deploy-preview-26405--kubernetes-io-master-staging.netlify.app/docs/concepts/services-networking/dns-pod-service/#namespaces-of-services

Reply:
@tengqm do you agree that the context for that sentence (“For example, consider a pod in a `test` namespace.” before “A query for data returns no results, because it uses the pod's test namespace.”) is sufficiently clear?

Reply:
@geoffcline rewriting into the active voice (“&hellip;when a container in that Pod queries DNS for&hellip;”) _might_ help?

Reply:
@sftim Yes. Such a change would help clarify the context.

Reply:
Maybe adding some constraints would be helpful.

DNS queries for services change based on namespace of the requesting pod.
DNS queries that don't specify a namespace are limited to the requesting pod's namespace.
A pod can access services in any namespaces by explicitly specifying the services' namespace.